modular_skyrat/modules/modular_items/lewd_items/code/lewd_clothing/ballgag.dm

```diff
@@ -1,211 +1,123 @@
+	/// Probability for alternative phrases to be used when speech is modified.
+	var/moans_alt_probability = 5
+
+	/// Sounds that the player makes when they try to speak.
 	var/list/moan_sounds = list('modular_skyrat/modules/modular_items/lewd_items/sounds/under_moan_f1.ogg',
 						'modular_skyrat/modules/modular_items/lewd_items/sounds/under_moan_f2.ogg',
 						'modular_skyrat/modules/modular_items/lewd_items/sounds/under_moan_f3.ogg',
-						'modular_skyrat/modules/modular_items/lewd_items/sounds/under_moan_f4.ogg') // Sounds that the proud wearer makes when they try to speak.
-	var/moan_volume = 70 // How loud are these moan-noises?
-	var/moan_volume_choking = 20 // How loud are these moan-noises when we're choking on it?
-	var/gag_customizability = GAG_COLORABLE
+						'modular_skyrat/modules/modular_items/lewd_items/sounds/under_moan_f4.ogg')
+	/// How loud the moan audio is
+	var/moan_volume = 50
+	/// How loud are these moan-noises if we're choking on it?
+	var/moan_volume_choking = 30
+
+	/// Whether the gag can be resized or not
+	var/resizable = FALSE
+	/// Options for resizing the gag
+	var/list/possible_gag_sizes = list("small", "medium", "large")
+	/// Where we are in the size list
+	var/size_list_position = 1
+	/// How big the gag is currently
 	var/gag_size = "small"
-	var/gag_color = "pink"
-	var/list/gag_colors // Populated after init, the available colors it can be
-	var/list/gag_sizes // Populated after init, the available sizes it can be (if any)
-	var/list/gag_choices
 
-// A ballgag, but it chokes! Also its a dick!
-/obj/item/clothing/mask/ballgag/phallic
-	name = "phallic ball gag"
-	desc = "Prevents the wearer from speaking, as well as making breathing harder."
-	icon_state = "chokegag"
-	inhand_icon_state = "blindfold"
-	icon = 'modular_skyrat/modules/modular_items/lewd_items/icons/obj/lewd_clothing/lewd_masks.dmi'
-	worn_icon = 'modular_skyrat/modules/modular_items/lewd_items/icons/mob/lewd_clothing/lewd_masks.dmi'
-	worn_icon_muzzled = 'modular_skyrat/master_files/icons/mob/clothing/mask_muzzled.dmi'
-	lefthand_file = 'modular_skyrat/modules/modular_items/lewd_items/icons/mob/lewd_inhands/lewd_inhand_left.dmi'
-	righthand_file = 'modular_skyrat/modules/modular_items/lewd_items/icons/mob/lewd_inhands/lewd_inhand_right.dmi'
-	time = 2
-	chokes_wearer = TRUE
-	moan_volume = 40
-	gag_customizability = GAG_COLORABLE_AND_SIZABLE
-
-// Basically just a kazoo that goes in the moan hole
-/obj/item/clothing/mask/ballgag/phallic/kazoo
-	name = "phallic kazoo gag"
-	desc = "While it doesn't prevent the wearer from breathing, it does make everything they say sound quite silly."
-	moans = list("BVVV...", "VRRRRH...", "VRRR...", "VZZZ...", "ZYYY...", "BZZZ...")
-	moans_alt = list("WHOOONK!", "VAAAAMP!", "BRRRVVV!")
-	moan_sounds = list('modular_skyrat/modules/modular_items/lewd_items/sounds/kazoo_1.ogg',
-						'modular_skyrat/modules/modular_items/lewd_items/sounds/kazoo_2.ogg',
```

Comment:
if nothing else references these sound files, they should be deleted too. unused sound files eat bandwidth like candy

Reply:
Ah you're right, entirely slipped my mind. Ty